joomla/joomla-cms
# Fix the default value.

### Summary of Changes

Fix the default value.

### Testing Instructions

- Enable the user- joomla plugin
- see that the pw fields are option when you try to create a new user.

### Expected result

pw fields are optional

### Actual result

pw fields are required

### Documentation Changes Required

none
